config/application.rb

```diff
@@ -90,7 +90,10 @@ class Application < Rails::Application
     config.action_cable.mount_path = '/ws/notifications'
 
     # Use yaml_unsafe_load for column serialization to handle Symbols
-    config.active_record.use_yaml_unsafe_load = true
+    config.active_record.use_yaml_unsafe_load = false
```

Comment:
I vote keeping this as `true`.

Reply:
@jrafanie can we have a discussion on this decision?

Reply:
💯 Agreed.  We discussed keeping this `true`.  I've opened https://github.com/ManageIQ/manageiq/issues/22795 with the list of workarounds I needed to get this to work in test.  I'll be splitting off some of the more recent commits in a followup for supporting yaml safe_load columns.